src/app/2025/(home)/_components/Stats/Stats.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  FaTrophy,
+  FaGraduationCap,
+  FaUsers,
+  FaHandsHelping,
+} from "react-icons/fa";
+
+import AnimatedCounter from "./AnimatedCounter";
+
+const data = [
+  {
+    icon: <FaTrophy size={32} className="text-black" />,
+    prefix: "$",
+    suffix: "+",
+    number: 2300,
+    title: "Prizes Awarded",
+    desc: "Plus: A greener future One tree planted every signup!",
```

Comment:
```suggestion
    desc: "Plus: A greener future one tree planted every signup!",
```
I'm confirming with WXP whether we're set to plant trees this year. Will get back to you